client/driver/fabfile.py

```diff
@@ -105,15 +108,24 @@ def run_oltpbench():
 
 @task
 def run_oltpbench_bg():
-    cmd = "./oltpbenchmark -b {} -c {} --execute=true -s 5 -o outputfile &".\
+    cmd = "./oltpbenchmark -b {} -c {} --execute=true -s 5 -o outputfile > "\
+          "/home/shulij/ottertune/client/driver/oltp.log 2>&1 &".\
           format(CONF['oltpbench_workload'], CONF['oltpbench_config'])
     with lcd(CONF['oltpbench_home']):  # pylint: disable=not-context-manager
         local(cmd)
 
 
 @task
 def run_controller():
-    cmd = 'sudo gradle run -PappArgs="-c config/sample_postgres_config.json -t 300"'
+    cmd = 'sudo gradle run -PappArgs="-c ' \
+          'config/sample_postgres_config.json -d output/postgres/" --no-daemon'
+    with lcd("../controller"):  # pylint: disable=not-context-manager
+        local(cmd)
+
+
+@task
+def stop_controller():
+    cmd = 'sudo ./stop_experiment.sh'
```

Comment:
You may want to remove stop_experiment.sh, and include the commands in this function.